book.rb

```diff
@@ -0,0 +1,11 @@
+class Book
+  attr_accessor :title, :author
+
+  attr_reader :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+end
```

Comment:
> Great job initialize `@rentals = [ ]` here 👍
- [ ] However, you should create a method for adding rentals and the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. (according to the requirements) 💯

> For Example 💡
```ruby
  def add_rental(person, date)
    Rental.new(date, self, person)
  end
```